commands/stats.js

```diff
@@ -0,0 +1,77 @@
+import * as bungie from '../bungieApi';
+import * as util from 'util';
+
+export const command = '/stats';
+export async function handler(payload) {
+  // do some stuff with the payload
+
+  const name = payload.text.length ? payload.text : payload.user_name;
+  const msg = {
+    "response_type": "ephemeral",
+    attachments: []
+  };
+
+  const member = [
+    ...await bungie.getPlayerID(1, name),
+    ...await bungie.getPlayerID(2, name)
+  ];
```

Comment:
nice, I'll take a look at lodash later.  tbh this was just a quick hack and I wanted to move all this into a lookup module that would decide what to do based on XBL/PSN tags etc..  There is a woeful lack of error checking anywhere at present :D